packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -428,6 +434,20 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
     return _ContextMenuLocation.left;
   }
 
+  // Constrain the size of the expanded child so that it does not go offscreen.
```

Comment:
nit: "go offscreen" should be rephrased to mention the safe area?

Reply:
I agree. I think 'safe area' should be used everywhere else in the documentation rather than 'offscreen'.